doc/deoplete.txt

```diff
@@ -793,6 +815,11 @@ is_bytepos
 		Because Vim script string uses byte position.  Python string
 		uses character position.
 
+					*deoplete-source-attribute-is_initialized*
+is_initialized
+		(Bool)				(Optional)
+		If it is True, |deoplete-source-attribute-on_init| is not called.
```

Comment:
It is needed?
It is just used internally.

Reply:
None of the sources listed in [wiki](https://github.com/Shougo/deoplete.nvim/wiki/Completion-Sources) seem to use it; removed.